sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpClientOptions.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import com.azure.core.util.CoreUtils;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * A set of AMQP client options.
+ */
+public class AmqpClientOptions {
```

Comment:
Do we have much of an idea if there is likely to be other properties that will eventually find their way into the options bag? I only ask because it can feel weird to introduce a new options type for a builder, rather than just put the single property in the options type directly on the builder. I can see that there is some nice symmetry here between what we do on the HTTP side, so I'm not objecting, just curious about thoughts for future growth plans.

Reply:
I have no idea about the future growth plans now. @conniey and @anuchandy do you have some future growth plans about this options? But I an idea about the specific method in builder. For example, in the `EventProcessorClientBuilder`, we have `partitionOwnershipExpirationInterval` and `loadBalancingUpdateInterval`, I think we can come up a new options extend `AmqpClientOptions` and put these properties in it. There are bunch of method in the builder, and if users don't have high-customized needs, these properties will be used the default value. Put these properties into a specific `AmqpClientOptions` can reduce the number of method in builder and it's more friendly for user with easy logic in their code (no need to understand each method in builder). I'm not sure does this meet the design guidelines?

Reply:
The `identifier` seems to be used only in EventHubs - is that correct? If yes then it's not good candidate for first "ampq option".

Reply:
Currently it is used in EventHubs, I think it will be used in Servicebus in next PR.

Reply:
we should be certain about cross services usage if we're introducing that. When is support in ServiceBus expected?

Reply:
@conniey Hi Connie, Kamil is asking when will we support ServiceBus to set identifier?

Reply:
I'd like to see it in Service Bus sooner rather than later.   It's quite helpful in the service logs to help correlate specific client instances.